alerter.c

```diff
@@ -8,24 +8,41 @@ int networkAlertStub(float celcius) {
     // Return 200 for ok
     // Return 500 for not-ok
     // stub always succeeds and returns 200
-    return 200;
+    if (celcius > 1000 && celcius < 0)
+       return 500;
+     else 
+       return 200;
 }
 
-void alertInCelcius(float farenheit) {
+int networkAlert(float celcius) {

```

Comment:
Stub is part of production code in the same file. This will get compiled and get shipped to field along with production code. Hence it is necessary to keep test code and production code separate in order to exclude test code for deployment.